src/xdocs/checks.xml

```diff
@@ -644,7 +655,7 @@
             <td>Checks lambda parameter names.</td>
           </tr>
           <tr>
-            <td><a href="checks/blocks/leftcurly.html#LeftCurly">LeftCurly</a></td>
+            <td><a href="config_blocks.html#LeftCurly">LeftCurly</a></td>
```

Comment:
This change looks wrong.

Reply:
done. reverted this change

Reply:
This is not done and not fully reverted.

Reply:
fixed